lib/src/base-styles/_nx-card.scss

```diff
@@ -14,91 +14,62 @@
   border-radius: $nx-border-radius;
   box-sizing: border-box;
   margin-bottom: $nx-spacing-md;
-  padding: $nx-spacing-l $nx-spacing-xxl;
-}
-
-.nx-card__call-out {
-  @include bold();
-
-  font-size: 32px;
-
-  .nx-icon--card-call-out {
-    font-size: 48px;
-  }
-
-  .nx-icon--card-call-out-xl {
-    font-size: 64px;
-  }
+  padding: $nx-spacing-l;
+  text-align: center;
 }
 
 .nx-card-container--row {
+  @include container-horizontal;
   display: flex;
-  -ms-flex-wrap: wrap;
   flex-wrap: wrap;
+  justify-content: space-between;
   margin-bottom: 0;
+}
 
-  + .nx-card-container--row {
-    margin-top: 0;
-  }
-
-  .nx-card {
-    @include container-vertical;
-    display: flex;
-    flex-direction: column;
-    -ms-flex-pack: justify;
-    justify-content: space-between;
-    margin-right: $nx-spacing-md;
-    text-align: center;
-    min-width: 230px;
-    max-width: 330px;
-  }
-
-  .nx-card__header {
-    @include container-vertical;
-    margin-bottom: $nx-spacing-md;
-  }
+.nx-card {
+  @include container-vertical;
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+  margin-right: $nx-spacing-md;
+  min-width: 230px;
+  max-width: 330px;
+}
 
-  .nx-card__call-out {
-    + .nx-card__content {
-      margin-top: $nx-spacing-md;
-    }
-  }
+.nx-card__header {
+  @include container-vertical;
+  margin-bottom: $nx-spacing-md;
+}
 
-  .nx-card__call-out--text-only {
-    + .nx-card__content {
-      margin-top: $nx-spacing-xs;
-    }
-  }
+.nx-card__content {
+  align-content: start;
+  align-items: start;
+  display: grid;
```

Comment:
What's the reason that this is a grid? I'd think it would work just as well as a normal block container using `container-vertical` and appropriate margins on its children

Reply:
> What's the reason that this is a grid? I'd think it would work just as well as a normal block container using `container-vertical` and appropriate margins on its children

"appropriate margins on its children" is the reason that I used `grid` it allows us to just use `row-gap` and not worry about applying that margin to every child object (which you know people will forget to do). I think that this will be easier for others implementing custom content.